pkg/http_endpoint/authentication/jwt_authentication.go

```diff
@@ -0,0 +1,106 @@
+package httpAuth
+
+import (
+	"github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+	"net/http"
+	"time"
+	"visualization-api/pkg/http_endpoint/common"
+)
+
+const contextJWTProperty = "AuthToken"
+
+// JWTIssuer
+const tokenIssuer = "visualization-api"
+
+// CustomClaims defines what data would be stored in jwt token
+type CustomClaims struct {
+	IsAdmin   bool   `json:"is_admin"`
```

Comment:
Please use isAdmin according to spec http://petstore.swagger.io/?url=https://raw.githubusercontent.com/zaletniy/visualization-api/master/doc/visualization-api-1.0.0.yaml